config/main.py

```diff
@@ -110,6 +110,23 @@ def set_interface_naming_mode(mode):
     user = os.getenv('SUDO_USER')
     bashrc_ifacemode_line = "export SONIC_CLI_IFACE_MODE={}".format(mode)
 
+    # Check alias 'key' presence in PORT dict
```

Comment:
Suggest change comment to: `# Ensure all interfaces have an 'alias' key in PORT dict`